mapknowledge/nposparql.py

```diff
@@ -37,6 +41,29 @@
 
 #===============================================================================
 
+NPO_OWNER = "SciCrunch"
+NPO_REPO = "NIF-Ontology"
+NPO_BRANCH = "neurons"
+NPO_DIR = "ttl/generated/neurons"
+NPO_SOURCE = f"https://raw.githubusercontent.com/{NPO_OWNER}/{NPO_REPO}/{NPO_BRANCH}/"
+NPO_PARTIAL_ORDER = "apinat-partial-orders.ttl"
+NPO_PARTIAL_ORDER_URL = f'{NPO_SOURCE}{NPO_DIR}/{NPO_PARTIAL_ORDER}'
+
+EXCLUDED_LAYERS = (
+    None,
+    'UBERON:0000010',      # peripheral nervous system
+    'UBERON:0000178',      # blood
+    'UBERON:0000468',      # multicellular organism
+    'UBERON:0001017',      # central nervous system
+    'UBERON:0001359',      # cerebrospinal fluid
+    'UBERON:0002318',      # spinal cord white matter
+    'UBERON:0003714',      # neural tissue
+    'UBERON:0005844',      # spinal cord segment
+    'UBERON:0016549',      # cns white matter
+)
```

Comment:
This is a duplicate of what is in `apinatomy.py`. Maybe there is a better place to put constants such as this (`anatomy.py`?) but having the same thing specified in more than one place is likely to create problems in the future...

Reply:
Yes, you are correct. I forgot to refer to the available one.